File: C_Programs/PrimeNumberUsingFunction.cpp
//create a function checkprimeno(int num) to check and display wheter the number is prime or not.

#include<stdio.h>
#include<conio.h>

void checkprimeno (int num)
{
	int cnt=2,rem;
	while (cnt<num)
	{
		 rem=num%cnt;
		 if(rem==0)
		 {
		 	break;
		 }
		 cnt++;	 
	}
	if(cnt==num)
	{
		printf("\n%d is a Prime Number",num);
	}
	else
	{
		printf("\n%d is not a Prime Number",num);
	}
	
}

int main()
{
	int num;
	printf("Enter any number :");
	scanf("%d",&num);
	checkprimeno(num);
	return 0;
}

File: Core_Java_Programs/PrimeNumberUsingFunction.java
//create a function checkprimeno(int num) to check and display wheter the number is prime or not.
import java.util.*;

public class PrimeNumberUsingFunction {
    public static void checkprimeno (int num)
    {
        int cnt=2,rem;
        while (cnt<num)
        {
            rem=num%cnt;
            if(rem==0)
            {
                break;
            }
            cnt++;
        }
        if(cnt==num)
        {
            System.out.println(num+" is a Prime Number");
        }
        else
        {
            System.out.println(num+" is not a Prime Number");
        }

    }

    public static void main(String[] args)
    {
        int num;
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter any number :");
        num=sc.nextInt();
        checkprimeno(num);
    }
}
